tcms/rpc/api/forms/testrun.py

```diff
@@ -18,6 +18,7 @@ class Meta:
     default_tester = UserField()
     start_date = DateTimeField()
     stop_date = DateTimeField()
+    # TODO should here the planned_stop and planned_start be added?
```

Comment:
yes, DateTimeField() is our custom class which specifies the input format for timestamps, see it's existing implementation. 

If you don't redefine the new fields in the form they will use whatever is standard for Django and that would lead to inconsistencies for consumers of this API (e.g. one field formatted in one way, another field formatted in a different way).